File: ManagingCompanies/LoadingACompanyWithACompanyID/LoadingACompanyWithACompanyID.cs
using System;
using RestSharp;
namespace LoadingACompanyWithACompanyID
{
    class LoadingACompanyWithACompanyID
    {
        static void Main(string[] args)
        {
            var client = new RestClient("https://api.apexchat.com/Services/ApexChatService.svc/companies/{companyID}");
            var request = new RestRequest(Method.GET);
            request.AddHeader("apexchat-company", "CompanyName");
            request.AddHeader("apexchat-username", "Username");
            request.AddHeader("apexchat-password", "Password");
            IRestResponse response = client.Execute(request);
            Console.WriteLine(response.Content);
        }
    }
}

File: ManagingCompanies/LoadingACompanyWithACompanyID/LoadingACompanyWithACompanyID.java
package ManagingCompanies.LoadingACompanyWithACompanyID;

import com.mashape.unirest.http.*;
import com.mashape.unirest.http.exceptions.UnirestException;

public class LoadingACompanyWithACompanyID {
    public static void main(String[] args) throws UnirestException {
        HttpResponse<String> response = Unirest
                .get("https://api.apexchat.com/Services/ApexChatService.svc/companies/{companyID}")
                .header("apexchat-company", "CompanyName").header("apexchat-username", "Username")
                .header("apexchat-password", "Password")
                .asString();
        System.out.println(response.getBody());
    }
}